modules/simple-payments/simple-payments.php

```diff
@@ -49,7 +49,9 @@ private function register_shortcode() {
 	public function init_hook_action() {
 		add_filter( 'rest_api_allowed_post_types', array( $this, 'allow_rest_api_types' ) );
 		add_filter( 'jetpack_sync_post_meta_whitelist', array( $this, 'allow_sync_post_meta' ) );
-		$this->register_scripts_and_styles();
+		if ( ! is_admin() ) {
```

Comment:
These scripts are not useful even if we're on wp-admin without Gutenberg? I.e. classic editor won't be missing them?

Reply:
The classic editor doesn't need them. These are just for the frontend to wire the actual pay button.